add right informations (#27)
code-rhapsodie/ezdataflow-bundle: README.md

Finally, click on the "Create" button.

# Rights

> If a no administrator user needs access to read dataflow interface, add policy `Setup / Administrate` and `eZ Dataflow / View` in the wanted role.

Reviewer: ```suggestion
If a non-administrator user needs read-only access to the dataflow interface, add the `Setup / Administrate` and `eZ Dataflow / View` policies in one of their roles.
```
